### DISCUSSION TITLE
Adding support for `elvish` shell

### DISCUSSION THREAD

**Discussion by mfts** [pkgxdev/pkgx#333] [Help]
asked via Discord by icidasset#5934

👋  I'm trying to add support for the https://elv.sh/ shell

How would one go about testing this (locally)? So far I got:
1. Install latest Deno
2. Set environment variables `TEA_PREFIX` and `VERSION`
3. `deno task install`

I'd assume I'd need to use `tea` as normal at that point.
But I'm getting the following:
```
error: Relative import path "hooks" not prefixed with / or ./ or ../ from "file:///Users/icid/Code/teaxyz-cli/src/app.ts"
```

**Comment by mxcl** [comment]
deno is apparently not finding our `deno.jsonc` config file. Is the user sure that they have the latest deno?

and more shells welcome for sure!

**Reply by icidasset** [reply]
Thanks! That was it, had to upgrade Deno.

**Reply by mxcl** [reply]
cool. Yeah the config-file is a relatively recent addition to deno.

LMK if you need more help. The additions for any shell are `app.magic.ts` and (if you want) in the installer script (@teaxyz/setup)

**Reply by icidasset** [reply]
@mxcl You prefer if I ask questions here, or on Discord?

**Reply by mxcl** [reply]
@icidasset hey sorry! Here is preferable. Though discord is more “manned”

**Reply by mxcl** [reply]
Though you got it in and I'm super glad :)
